cmd/integration-test/ssl.go

```
@@ -123,7 +123,10 @@
 type sslMultiReq struct{}
 
 func (h *sslMultiReq) Execute(filePath string) error {
-	ts := testutils.NewTCPServer(&tls.Config{}, defaultStaticPort, func(conn net.Conn) {
+	ts := testutils.NewTCPServer(&tls.Config{
+		MinVersion: tls.VersionSSL30,
```

golangci-lint (1.62.2)

SA1019: tls.VersionSSL30 has been deprecated since Go 1.13 because it shouldn't be used: SSLv3 is cryptographically broken, and is no longer supported by this package. See golang.org/issue/32716. 

(staticcheck)